scripts/dependencies/tools.ts

```diff
@@ -4,12 +4,23 @@ import path from 'path';
 
 import semver from 'semver';
 
+import { rcompareVersions } from './lima';
 import {
-  download, downloadZip, downloadTarGZ, getResource, DownloadOptions, ArchiveDownloadOptions,
+  ArchiveDownloadOptions,
```

Comment:
We should see if we can get a linter to do your preferred style of imports…

I think we are already using [https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md#named](`eslint-plugin-import`) so we can just turn the relevant option on (in a different PR)?